### ISSUE TITLE
UI freezes when changing Background Image - force quit required

### ISSUE THREAD

**Issue by dunxd** [jgraph/drawio-desktop#789]
### Preflight Checklist

* [x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio-desktop/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

**Describe the bug**
UI freezes when changing Background Image - force quit required

**To Reproduce**
1. Open diagram that already has a background image
2. Click on 'File > Page Setup'
3. Click on Image: Change button
4. Put cursor into image path box - freezes instantly.  None of the buttons work anymore. Unable to get back to drawing.  Need to use Force Quit to stop the app.  Menu items still available, but Quit draw.io doesn't work.

**Expected behavior**
Allow editing of the Image path

**Screenshots**

**draw.io version (In the Help->About menu of the draw.io editor):**

- draw.io version [15.7.3] - note that draw.io > about draw.io menu item goes to draw.io home page, rather than showing details about the version installed. 

**Desktop (please complete the following information):**
 - OS: [macos 12.0.1 Monterey]

**Additional context**
No additional context

**Comment by dunxd** [comment]
Same issue occurs when adding a background image to a new diagram - after dragging the image, the UI stops responding.

**Comment by dunxd** [comment]
Does not occur with 15.7.3 on Windows 10

**Comment by davidjgraph** [comment]
I cannot repeat on ARM, is this intel? Does it happen regardless of the background image used? I tried https://www.pngfind.com/pngs/m/503-5032998_ortler-traverse-example-png-transparent-png.png

**Comment by dunxd** [comment]
Should have mentioned, I'm on an Intel Mac.

I have now tested with some different images, and it seems to be the issue with large dimension images - 2246 x 2068 is the one I used.  It is only 1.1 Mb filesize.

I had previously been able to use this image in a background no problem - the issue arose when I tried to edit a drawio file that already had this background image, as well as adding the same image as a background image into a new diagram.

I made a smaller version of the file - 862 * 794 px / 238kb.  This works, but there is a very long pause during which the app is unresponsive and shows no signs of life. 

With the larger image, it was longer than 5 minutes before I decided to force quit.
